luarules/gadgets/pve_areahealers.lua

```diff
@@ -101,7 +79,7 @@ for unitDefID, unitDef in pairs(UnitDefs) do
 end
 
 function gadget:UnitCreated(unitID, unitDefID, unitTeam)
-    if healersTable[unitDefID] and (unitTeam == scavengerAITeamID or unitTeam == raptorsAITeamID) then
+    if healersTable[unitDefID] and (unitTeam == pveTeamID or unitTeam == pveTeamID) then
```

Comment:
You're checking the same condition twice

Reply:
sorry bad usage of search and replace lol